packages/app/obojobo-document-engine/documents/empty.xml

```diff
@@ -90,6 +90,22 @@
 					</Page>
 				</scoreAction>
 			</scoreActions>
+			<triggers>
```

Comment:
Minor style suggestion, not an expert on this but you could remove the empty value tag to simplify this a bit:
```xml
<triggers>
	<trigger type="onStartAttempt">
		<actions>
			<action type="nav:lock" />
		</actions>
	</trigger>
	<trigger type="onEndAttempt">
		<actions>
			<action type="nav:unlock" />
		</actions>
	</trigger>
</triggers>
```